package.json

```diff
@@ -4,9 +4,9 @@
     "version": "0.0.0",
     "devDependencies": {
         "@nomicfoundation/hardhat-chai-matchers": "^1.0.6",
-        "@nomicfoundation/hardhat-ethers": "^3.0.7",
-        "@nomiclabs/hardhat-etherscan": "^3.1.7",
-        "@nomiclabs/hardhat-web3": "^2.0.0",
+        "@nomicfoundation/hardhat-ethers": "^3.1.0",
```

Comment:
hardhat-ethers v3 requires hardhat v2 and ethers v6.
There's also hardhat-ethers v4, but it requires hardhat v3.
ethereum-contracts package still uses ethers v5, thus has an overriding import of hardhat-ethers v2 which depends on ethers v5.